tests/test_lfw_format.py

```diff
@@ -143,6 +143,38 @@ def test_can_save_and_load_with_landmarks(self):
 
             compare_datasets(self, source_dataset, parsed_dataset)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_save_and_load_with_only_landmarks(self):
+        source_dataset = Dataset.from_iterable(
+            [
+                DatasetItem(
+                    id="name0_0001",
+                    subset="test",
+                    media=Image(data=np.ones((2, 5, 3))),
+                    annotations=[
+                        Points([0, 4, 3, 3, 2, 2, 1, 0, 3, 0], label=0),
+                    ],
+                ),
+            ],
+            categories=["name0"],
+        )
+
+        target_dataset = Dataset.from_iterable(
+            [
+                DatasetItem(
+                    id="name0_0001",
+                    subset="test",
+                    media=Image(data=np.ones((2, 5, 3))),
+                ),
+            ],
+            categories=["name0"],
+        )
```

Comment:
I don't think this behavior in the test is right. From the format viewpoint, the input annotations are incomplete, and it doesn't seem we can adequately support such cases in the format, so the best approach would be either to complete the missing annotations, if possible, or raise a clear error message otherwise. 

My considerations on this specific case with CVAT export are [here](https://github.com/opencv/cvat/issues/5100#issuecomment-1369516259). Let's try to make the format clearly usable from CVAT, Datumaro seem to be correct at this point.